tests/src/shader-lab/ShaderLab.test.ts

```
@@ -188,6 +188,7 @@ describe("ShaderLab", () => {
   });
 
   it("engine shader", async () => {
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)